master/internal/rm/resource_manager_iface.go

```diff
@@ -62,6 +62,7 @@ type ResourceManager interface {
 	ResolveResourcePool(
 		ctx actor.Messenger,
 		name string,
+		workspace,
```

Comment:
workspaceID int?

Reply:
In Go, you can omit the type if the following arguments are the same type - in this case I have `workspace int, slots int` which I can shorten to `workspace, slots int`